documentation/staging/content/quickstart/create-domain.md

```diff
@@ -2,57 +2,50 @@
+   - Use the following command to apply the sample domain resource:
 
-    For example, assuming you named your copy `my-inputs.yaml`:
+       ```shell
+       $ kubectl apply -f https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml
+       ```
 
-    ```shell
-    $ cd kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image
-    ```
-    ```shell
-    $ ./create-domain.sh -i my-inputs.yaml -o /<your output directory> -u <username> -p <password> -e
-    ```
-    {{% notice note %}}You need to provide the same WebLogic domain administrator user name and password in the `-u` and `-p` options
-    respectively, as you provided when creating the Kubernetes Secret in Step 2.
-    {{% /notice %}}
+    - **NOTE**: Instead of running the previous `kubectl apply` command, you can download the WLS Domain YAML file to `/tmp/quickstart/quick-start-domain-resource.yaml` or similar, using the following command, and then apply the file using `kubectl apply -f /tmp/quickstart/quick-start-domain-resource.yaml`:
 
-    For the detailed steps that the `create-domain.sh` script performs, see [Domain Home In Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}).
+      ```shell
+      $ curl -m 120 -fL https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml -o /tmp/quickstart/quick-start-domain-resource.yaml
+      ```
+      This domain resource references a WebLogic Server installation image, the secrets you defined, and a sample "auxiliary image," which contains traditional WebLogic configuration and a WebLogic application.
+         - To examine the domain resource, click [here](https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/quick-start-domain-resource.yaml).
+         - For detailed information, see [Domain resource]({{< relref "/managing-domains/domain-resource.md" >}}).
 
+   {{% notice note %}}
+   The Quick Start guide uses a WebLogic Server version 12.2.1.4 General Availability (GA) image. GA images are suitable for demonstration and development purposes _only_ where the environments are not available from the public Internet; they are **not acceptable for production use**. In production, you should always use CPU (patched) images from [OCR]({{< relref "/base-images/ocr-images.md" >}}) or create your images using the [WebLogic Image Tool]({{< relref "/base-images/custom-images#create-a-custom-base-image" >}}) (WIT) with the `--recommendedPatches` option. For more guidance, see [Apply the Latest Patches and Updates](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/standalone/weblogic-server/14.1.1.0&id=LOCKD-GUID-2DA84185-46BA-4D7A-80D2-9D577A4E8DE2) in _Securing a Production Environment for Oracle WebLogic Server_.
```

Comment:
The Quick Start guide uses a 
-->
The Quick Start guide's sample domain resource references a